bedrock/settings/__init__.py

```diff
@@ -102,6 +102,13 @@
     DEV_LANGUAGES = PROD_LANGUAGES
     LANGUAGE_CODE = "en"  # Pocket uses `en` not `en-US`
 
+    COOKIE_CONSENT_SCRIPT_SRC = (
+        "https://cdn.cookielaw.org/scripttemplates/otSDKStub.js"
+        if DEV
+        else "https://cdn.cookielaw.org/consent/a7ff9c31-9f59-421f-9a8e-49b11a3eb24e/otSDKStub.js"
+    )
+    COOKIE_CONSENT_DATA_DOMAIN = "a7ff9c31-9f59-421f-9a8e-49b11a3eb24e-test" if DEV else "a7ff9c31-9f59-421f-9a8e-49b11a3eb24e"
```

Comment:
👍

Reply:
Do we need to add these in mozmeao www-config as well? Do we need to notify SRE to update anything the GCP settings? cc: @stevejalim

Reply:
These settings only need to change depending on if Dev=True or not, so shouldn’t need any env specific variables set, I think?